src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/BuilderInfo.java

```diff
@@ -135,35 +136,19 @@ public String getSha1() {
   }
 
   public String getTestCompileResponse() {
-    return testCompileResponse;
-  }
-
-  public void setTestCompileResponse(String testCompileResponse) {
-    this.testCompileResponse = testCompileResponse;
-  }
-
-  public String getSolutionCompileResponse() {
-    return solutionCompileResponse;
+    return Joiner.on("\r\n").join(testCompileResponse);
```

Comment:
Now we have java8 I think I prefer: testCompileResponse.stream().collect(Collectors.joining("\r\n")) - i suspect guava will eventually remove joiner. I _think_ that this is safe on a Concurrent collection.